pkg/manager/agent.go

```diff
@@ -68,11 +68,7 @@ func (a *AgentDaemonSet) Create(image string, managerURL string) error {
 				},
 				Spec: corev1.PodSpec{
 					NodeSelector: a.sbm.getNodeSelector(),
-					Tolerations: []corev1.Toleration{
-						{
-							Operator: corev1.TolerationOpExists,
-						},
-					},
+					Tolerations:  a.sbm.getTaintToleration(),
```

Comment:
Just a note: This will change the current behavior if no taint tolerations are specified since it was to tolerate all taints.